plugins/publish-changes/admin/src/components/LatestRun.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from "react";
+import { request } from "strapi-helper-plugin";
+import pluginId from "../pluginId";
+
+const LatestRun = () => {
+  const [latestRun, setLatestRun] = useState({});
+
+  useEffect(() => {
+    async function fetchLatestRun() {
+      try {
+        const res = await request(
+          `${strapi.backendURL}/${pluginId}/runs/latest`
+        );
+        setLatestRun(res);
+      } catch (e) {
+        strapi.notification.toggle({
+          type: "warning",
+          message: e.response.payload.error,
+        });
+      }
+    }
+
+    fetchLatestRun();
+
+    const interval = setInterval(fetchLatestRun, 10000);
```

Comment:
Maybe use `setTimeout` after the previous request, so requests don't stack up if they take long for some reason

Reply:
I updated this, please check the new implementation